webpack/tools/testSetup.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+// file probably for enzyme, remove
+
```

Comment:
This file was committed by accident. There are no tests yet. I plan to do a separate PR for adding testing.